UI/SidePanel.cpp

```diff
@@ -1136,12 +1145,25 @@ void SidePanel::PlanetPanel::RefreshPlanetGraphic() {
         m_planet_graphic->Play();
     } else if (planet->Type() < NUM_PLANET_TYPES) {
         int planet_image_sz = PlanetDiameter(planet->Size());
+        planet_graphic_height = GG::Y(MaxPlanetDiameter());
         GG::Pt planet_image_pos(GG::X(MaxPlanetDiameter() / 2 - planet_image_sz / 2 + 3),
                                 GG::Y(MaxPlanetDiameter() / 2 - planet_image_sz / 2));
         m_rotating_planet_graphic = new RotatingPlanetControl(planet_image_pos.x, planet_image_pos.y,
                                                               m_planet_id, m_star_type);
         AttachChild(m_rotating_planet_graphic);
     }
+
+    // add blockade graphic
+    if (!GetSupplyManager().SystemHasFleetSupply(planet->SystemID(), planet->Owner()) && (!planet->Unowned())) {
```

Comment:
not having fleet supply doesn't necessarily mean a planet is blockaded, such as if there is no supply source that could read it in any case...

Reply:
At most one empire supplies each system. If empire B's supply range is bigger in a system, it will overpower the empire A's supply range, and A won't be able to supply fleets there (unless allied). So far as I'm aware, this is not considered a "blockade".

Reply:
> So far as I'm aware, this is not considered a "blockade"... although perhaps it should be?

Although I could see some validity to the idea of splitting the "blockade" concept into "fleet blockade" and "supply blockade", where the former is always accompanied by the latter but the latter might happen without the former, that seems to me to get a bit over-complicated.  I think I'd rather just refer to this situation as a "supply-contest-tie" or somesuch.  But I wouldn't object to that situation still leading to the planet-blockade icon being shown, it's just that it's tooltip then should be a bit more specific/clear.

> I'm also not sure what happens when a planet just has 0 or less than 1 supply range. Can it supply fleets in system in that case? If not, it probably shouldn't be considered a blockade, as nothing is being blocked.

It's not so common to even encounter a clear situation where this can be tested.  I am pretty sure, though, that  few weeks ago I did see a situation where I could test this, at either a Huge planet colony or a colony planet with elliptical orbit special or accretion disk system special, and my ships **did** still resupply there, at least if I stopped.  (I'll acknowledge, the scope of either my testing or memory is leaving something to be desired.)  However it's also my recollection that at GasGiant outposts they would not-- if I am not mis-remembering that then perhaps there is some difference between colonies and outposts?  I will try to do some more thorough testing sometime.